interface/docker-based-rock-ons/netdata_official.rst

```diff
@@ -49,14 +50,14 @@ First, navigate to *System* > *Groups*, and search for the **docker** group in
 the bottom table (*Other system groups*). In the example below, the **docker**
 group has a :code:`GID` of :code:`476`.
 
-.. image:: ../images/docker-based-rock-ons/netdata_docker_group.png
+.. image:: /images/interface/docker-based-rock-ons/netdata_docker_group.png
```

Comment:
Thanks for standardising how we reference the "/images" directory. I'd not noticed the disparate use.